crypto/fipsmodule/kem/kem.c

```diff
@@ -306,3 +308,45 @@ int KEM_KEY_set_raw_key(KEM_KEY *key, const uint8_t *in_public,
 
   return 1;
 }
+
+int KEM_KEY_set_raw_keypair_from_seed(KEM_KEY *key, const CBS *seed) {
+  if (key == NULL || seed == NULL || key->kem == NULL) {
```

Comment:
Should we also confirm here that `key->public_key` and `key->secret_key` are both NULL -- otherwise the logic below would overwrite these pointers and leak memory?

Reply:
Added this check in [69a4ec3](https://github.com/aws/aws-lc/pull/2707/commits/69a4ec3331287f892fc761050034093517e6f4bb)